nevergrad/instrumentation/multivariables.py

```diff
@@ -189,6 +189,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.keywords: Tuple[Optional[str], ...] = ()
         self.variables: List[Variable] = []
         self._set_args_kwargs(args, kwargs)
+        self._is_noiy: Bool = false
```

Comment:
`False` not `false`  :p 
What is the aim though?
- notifying that the instrumentation is noisy? This already exists (there's a "noisy" attribute in instrumentations, which will be renamed in the next version btw)
- notifying that the function is noisy? where will it be used? If so, I think this is somehow related to the `UnorderedDiscrete` variable I would say, in the next version of instrumentation I'm trying to merge all these into some kind of "descriptor" class giving information to the function.

Reply:
This is about the fact that the objective function is noisy.
But this is not related to unordered discrete --- an objective function can be ordered and noise-free, unordered and noisy, or the domain can be continuous and the objective function can be noisy, etc.

Reply:
in the code: "noisy=any(v.noisy for v in self.variables)  so this refers to noise induced by the instrumentation, not to noise in the objective function.